src/main/java/org/checkerframework/checker/noliteral/NoLiteralChecker.java

```diff
@@ -0,0 +1,22 @@
+package org.checkerframework.checker.noliteral;
+
+import org.checkerframework.common.basetype.BaseTypeChecker;
+import org.checkerframework.framework.source.SuppressWarningsKeys;
+
+/** The NoLiteral Checker enforces that literals don't go where they don't belong */
+@SuppressWarningsKeys("noliteral")
+public class NoLiteralChecker extends BaseTypeChecker {
+
+  /**
+   * Overridden so that unchecked code defaults are always used for bytecode, because we only want
```

Comment:
Give the specification for the method.  Say what it does rather than starting off with an obvious fact about the implementation (that it is overridden),  When you write something, readers will assume it is important, and when it's the first thing you write, readers should assume it is the most important thing.  If that is not the case, the writing is confusing.